CMakeLists.txt

```diff
@@ -786,6 +786,14 @@ if(BOUT_BUILD_EXAMPLES)
   add_subdirectory(examples)
 endif()
 
+##################################################
+# Documentation
+
+option(BOUT_BUILD_DOCS "Build the documentation" ON)
```

Comment:
Would it make sense to only default to ON if it can be build?
Most users will not need it ...

Reply:
Fair point that most users won't need it, as the docs are available online. Might be best to default `OFF` then.